enterprise/app/controllers/api/v1/accounts/audit_logs_controller.rb

```diff
@@ -21,6 +21,14 @@ def show
   private
 
   def fetch_audit
-    @audit_logs = Current.account.associated_audits.order(created_at: :desc)
+    @audit_logs = if audit_logs_enabled?
+                    Current.account.associated_audits.order(created_at: :desc)
+                  else
+                    Current.account.associated_audits.none
```

Comment:
@scmmishra You cannot return an empty array as there would be a chained request after this method. Use `Model.none` to return chainable relation with zero records